modules/formatter/src/util/string_ops.scala

```diff
@@ -121,14 +124,6 @@ object string_ops {
         else (acc + delimiter + indentation + line, isFirst)
       }
       ._1
-    if (
```

Comment:
I've removed this and instead rely on a count of element to decide whether to break
This is because it wasn't playing nice with comments (because most of the time, the value included comment and newLines and this would remove all of them and put everything in a line)